assets/js/modules/sign-in-with-google/components/setup/SetupForm.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * SIgn In With Google Setup form.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useSelect } from 'googlesitekit-data';
+import StoreErrorNotices from '../../../../components/StoreErrorNotices';
+import { MODULES_SIGN_IN_WITH_GOOGLE } from '../../datastore/constants';
+import ClientIDTextField from '../common/ClientIDTextField';
+import Button from '../../../../googlesitekit/components-gm2/Button';
+import { CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
+import ExternalIcon from '../../../../../svg/icons/external.svg';
+
+export default function SetupForm() {
+	const siteName = useSelect( ( select ) =>
+		select( CORE_SITE ).getSiteName()
+	);
+	const homeURL = useSelect( ( select ) => select( CORE_SITE ).getHomeURL() );
+
+	return (
+		<form>
```

Comment:
This probably doesn't have to be `<form>` because we don't have `onSubmit` handler here. Let's replace it to be a div to make sure that it can't be submitted by accident. Also let's use `className="googlesitekit-sign-in-with-google-setup__form"` for consistency with other setup forms.

Reply:
Thanks, that's make sense, I kept for consistency as `form` but makes more sense to be `div`